files/default/dashboards/hops/onlinefs.json

```diff
@@ -747,7 +747,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum by (topic) (\nrate(onlinefs_row_topic_counter_total[30s])\n)",
+          "expr": "sum by (topic) (\nrate(onlinefs_row_topic_counter_total{topic=\"$topic\"}[30s])\n)",
```

Comment:
The idea of this dashboard was to show the number of messages coming in across all feature groups. Can you please revert this change?